pagopa/io-app
# chore(IT Wallet): [SIW-1313] Add IT Wallet trial system status

## Short description
This PR allows the app to know if the user is enabled to use the IT Wallet experimental features

## List of changes proposed in this pull request
- Fixed `trial_system` typo in type definitions script
- Added IT Wallet trial system status read at app boot 
- Added IT Wallet trial system playgrounds

## How to test
With the `io-dev-api-server`, go to **Profile > Playgrounds > IT Wallet > Trial**. You should be able to enable, refresh and reset the IT Wallet trial system.

## Preview

https://github.com/pagopa/io-app/assets/6160324/1c638850-2fda-42b9-9c8b-6a53b137dce6
